test/distributed/fsdp/test_fsdp_fx.py

```diff
@@ -0,0 +1,118 @@
+# Owner(s): ["oncall: distributed"]
+
+from typing import Any
+
+import torch
+from torch.distributed.fsdp._symbolic_trace import _init_execution_info, _patch_tracer
+from torch.testing._internal.common_fsdp import FSDPTest
+from torch.testing._internal.common_utils import (
+    instantiate_parametrized_tests,
+    run_tests,
+)
+
+
+class Model(torch.nn.Module):
+    def __init__(self) -> None:
+        super().__init__()
+        self.weight1 = torch.nn.Parameter(torch.randn(6, 6))
+        self.weight2 = torch.nn.Parameter(torch.randn(6, 6))
+        self.weight_unused = torch.nn.Parameter(torch.randn(2, 2))
+        self.layer0 = torch.nn.Linear(6, 6)
+        self.layer1 = torch.nn.Linear(6, 6, bias=False)
+        self.layer2 = torch.nn.Sequential(
+            torch.nn.Linear(6, 3, bias=False),
+            torch.nn.ReLU(),
+            torch.nn.Linear(3, 6, bias=False),
+        )
+        self.relu = torch.nn.ReLU()
+
+    def forward(self, x: Any, flag: bool):
```

Comment:
nit: Similar comment as below about renaming `flag` to something more descriptive.